shop/settings.py

```diff
@@ -121,4 +121,5 @@
 
 STATIC_URL = '/static/'
 STATICFILES_DIRS=[os.path.join(BASE_DIR,'static')]
-CRISPY_TEMPLATE_PACK='bootstrap4'
\ No newline at end of file
+CRISPY_TEMPLATE_PACK='bootstrap4'
+LOGIN_REDIRECT_URL='user_home'
```

Comment:
this is the URL to redirect to after logging in with Django's authentication system